app/notification/model/notification.py

```diff
@@ -19,3 +24,28 @@ def from_json(data: dict):
             content=data.get("content"),
         )
         return notification_data
+
+    @staticmethod
+    def process_data(json_data):
+        data = Notification.from_json(json_data)
+        match data.template_type:
+
+            case NotifyConstants.TEMPLATE_TYPE_MAGIC_LINK:
+                current_app.logger.debug(
+                    "Second step - connect data with MAGIC_LINK service class"
+                    " to  process contents"
+                )
+                return MagicLink.from_json(data)
+
+            case "APPLICATION_RECORD_OF_SUBMISSION":
+                current_app.logger.info(
+                    "Second step - connect data with APPLICATION service"
+                    " class to  process_data"
+                )
+                return Application.from_json(data)
+
+            case "NOTIFICATION" | "REMINDER" | "AWARD":
```

Comment:
Can we just change this to `case _` and remove the block below?

Reply:
will do